src/Microsoft.Framework.FileSystemGlobbing/Internal/PathSegments/LiteralPathSegment.cs

```diff
@@ -7,7 +7,9 @@ namespace Microsoft.Framework.FileSystemGlobbing.Internal.PathSegments
 {
     public class LiteralPathSegment : IPathSegment
     {
-        public LiteralPathSegment(string value)
+        private readonly StringComparison _comparisonType;
+
+        public LiteralPathSegment(string value, StringComparison comparisonType)
```

Comment:
Need `StringComparison` here: https://github.com/aspnet/FileSystem/blob/trdai/dnx1645/src/Microsoft.Framework.FileSystemGlobbing/Internal/PathSegments/WildcardPathSegment.cs#L39

Know a way to get `StringComparison` from `StringComparer` easily?